### PR TITLE
Fix Console Title test when running on RS2

### PR DESCRIPTION
**Pull request by tarekgh**

### CODE REVIEW

**Comment by tarekgh** [comment]
CC @danmosemsft

File: src/System.Console/tests/WindowAndCursorProps.cs [hunk 1]
```diff
@@ -154,7 +154,16 @@ public static void Title_Set_Windows(int lengthOfTitle)
         {
             string newTitle = new string('a', int.Parse(lengthOfTitleString));
             Console.Title = newTitle;
-            Assert.Equal(newTitle, Console.Title);
+
+            if (newTitle.Length > 513 && PlatformDetection.IsWindowsRedStone2)
+            {
+                // RS2 has a bug when getting the windnow title when the title length is longer than 513 character
```
Comment (mellinoe) [inline src/System.Console/tests/WindowAndCursorProps.cs]: typo: ~windnow~ window

**Comment by tarekgh** [comment]
> Should we file an issue to delete this workaround at some point? I guess we will stop caring about this when everyone has stopped using the problematic builds.

Windows team decided not to fix this for RS2. if we are going to enusre in the future we'll not run on RS2, then yes I can track that with issue. But I don't think this will be the case. we'll live with that for good.

**Comment by tarekgh** [comment]
Thanks @EricaMo for catching the typo :-)

**Pull request merged**